README.md

```diff
@@ -1,83 +1,40 @@
 [![Build Status](https://travis-ci.org/DataDog/dd-agent.svg?branch=master)](https://travis-ci.org/DataDog/dd-agent)
 
-The Datadog Agent faithfully collects events and metrics and brings
-them to [Datadog](https://app.datadoghq.com) on your behalf so that
-you can do something useful with your monitoring and performance data.
+# Important note
 
-You're looking at the source code right now. We provide a number of
-pre-packaged binaries for your convenience for both [Agent 5](https://app.datadoghq.com/account/settings?agent_version=5#agent) and [Agent 6]((https://app.datadoghq.com/account/settings#agent)). 
+This repository contains the source code for the Datadog Agent until major version 5.
+Although still supported, no major feature is planned for this release line and we
+encourage users and contributors to refer to the new Agent codebase, introduced
+with the release of version 6.0.0 and tracked in
+[a different git repository](https://github.com/DataDog/datadog-agent).
 
-See the [Datadog Agent Repo](https://github.com/DataDog/datadog-agent) for the source code for Agent 6. 
+## Changes
 
-Note: this repository does not contain the sources of the Trace Agent that
-collects traces for the APM feature. If you choose the installation from source
-or the Windows and Mac OS X builds you will need to install the Trace Agent
-separately. See
-[dd-trace-agent](https://github.com/DataDog/datadog-trace-agent) for
-instructions.
+Please refer to the [Change log](https://github.com/DataDog/dd-agent/blob/master/CHANGELOG.md)
+for more details about the changes introduced at each release.
 
-# [Change log](https://github.com/DataDog/dd-agent/blob/master/CHANGELOG.md)
+## Integrations
 
-# How to contribute code
```

Comment:
you've removed this section entirely, is that to indicate that we don't accept contributions anymore? (we should make that clear in that case)

Reply:
Yep that was the goal. I'll rephrase keeping the basic info without the emphasis

Reply:
Is there a reason we wouldn't accept a bug or security fix? May be worth emphasizing that all feature work should happen in Agent 6, and only PRs for large impact bug fixes and security issues should be submitted for Agent 5.